docs/flows/flows.puml

```diff
@@ -130,7 +134,27 @@ alt#Gold #DarkSeaGreen trace
   bitdaoadmin <-- bitdaogovernance: TxHash
 end
 
+newpage
+
+autonumber "Upgrades:[00]"
+
+== Upgrades ==
+
+alice -> forum: create forum upgrade proposal
+alice <- guarantor: positive/negative feedback
+
+community -> community: delegate role based votes
+
+alice -> bitdaogovernance: if (roles) submit upgrade proposal
+community -> bitdaogovernance: roles voting
+
+bitdaoadmin -> bitdaogovernance: show roles voting result
+bitdaoadmin <-- bitdaogovernance: pass/fail
```

Comment:
suggestion: `pass/fail` -> `pass`

In sequence diagrams there are a few ways to deal with multiple outcomes, the popular ones I've seen:
1) Seperate diagram: A different diagram for each possible outcome for each branch (this leads to many diagrams, if done exhaustively).
2) Sub-diagram: Some UML tools support nesting diagrams (unsure whether plantuml does), where each branch is a sub-diagram.
3) Notes: Some UML tools support adding notes (unsure whather plantuml does), these can be used to cover broad outcomes, whose documenting un UML brings little value e.g. proposal fails, flow exits.
4) Only focus on critical / high value flows:  Exclude explicitly mentioning/covering branches whose documentation is implicitly clear e.g. proposal fail, the flow exits.

Reply:
We can omit fail stage, because, well we can assume that nothing happens next if failed, unless it specified.